### ISSUE TITLE
authorisation method needs manual option

### ISSUE THREAD

**Issue by nutpantz** [deltachat/deltachat-android#1019]
With Latest update I have had to trouble shoot 4 friends to disable oauth2 switching to automatic and messing EVERYTHING up. Had to walk everybody through turning off oauth2 for BOTH delta chat AND Google. (And no, no one used oauth2 before)
They're really needs to be a force oauth2 off option, not automatic which seems to "magically" turn it self on sometimes.

**Comment by r10s** [comment]
if you answer "cancel" on first setup, oauth2 is not enabled. later on, you can disable it. 

there is a summary for that at: https://support.delta.chat/t/oauth/568

i've just tried out both methods, closing this issue therefore for now.

**Comment by nutpantz** [comment]
Are you saying that EVERYONE who updates and has oauth2 mess up with Google has to completely uninstall delta chat, lose all their contacts and everything else just to get it to work with Gmail?
Maybe you could remove oauth2 until it works and does not mess things up on updates. Or at least have a manual "force off" option.

**Comment by r10s** [comment]
no need to uninstall delta to disable oauth. just follow the "existing delta installation" on https://support.delta.chat/t/oauth/568

**Comment by nutpantz** [comment]
Maybe you missed that all that had been done before and when delta chat updated it switched to oauth2, Google switched to oauth2 "all by them selves with no actions from the user" and then failed to access the account, at which point I had to AGAIN walk 4 users through switching EVERYTHING (Google and delta chat) to less secure apps and user password..(which is how it was set up before the DELTA CHAT UPDATE).

**Comment by r10s** [comment]
@nutpantz not sure if i get the point :)

delta chat did not update or upgrade any gmail accounts automatically from less-secure to oauth2. oauth2 was only offered for new/update accounts after some user interaction.

but yes, the current inactive oauth2 is annoying, but there's not much we can do about that. these are the problems with centralized systems ;)

**Comment by nutpantz** [comment]
You don't get the point.
Delta chat was working fine on 4 peoples devices with user password and with gmail on less secure apps . After Delta chat updated all 4 people were switched over to oauth2, with incorrect passwords in delta chat and gmail had turned off less secure app access.. I set up all 4 devices and gmail accounts for these people.I know exactly how they were running before the update. (They did not all update at the same time) when I heard the 4th had the exact same issue needing the exact same fix. (Switch delta chat to automatic authorization and gmail back to less secure apps and then check delta chat user password is right) I wrote my bug report..
Every one got their update from fdroid.(once again not all at the same time)
